src/components/Hero/Hero.module.css

```diff
@@ -1,3 +1,139 @@
 .hero {
```

Comment:
Please check border radius and paddings
<img width="445" height="444" alt="Screenshot 2025-09-27 at 11 15 36" src="https://github.com/user-attachments/assets/1d3fa7a5-3c6e-4590-a708-46c166c27c0f" />